Server/src/main/webapp/resources/js/view.js

```diff
@@ -0,0 +1,109 @@
+class View {
+    _currentUser;
+    _postTemplate = document.getElementById('twit-template');
+    _postContainer = document.getElementById('container');
+    _postCreation = document.getElementById('twitAdd');
+    _header = document.getElementById("header");
+
+    _fillPostData(postTemplate, data) {
+        if (this._currentUser !== data.author) {
+            postTemplate.querySelector('[class = "twit-buttons"]').setAttribute('style', 'display: none')
+        }
+        postTemplate.firstElementChild.setAttribute('id', data.id);
+        if (data?.photoLink != null) {
+            postTemplate.querySelector('[data-target = "photoLink"]').setAttribute('src', data.photoLink);
+        } else {
+            postTemplate.querySelector('[data-target = "photoLink"]').setAttribute('style', 'display: none');
+        }
```

Comment:
Minor. Empty line after if. Everything else from code style in this file is good